app/assets/javascripts/projects/index.js

```diff
@@ -478,9 +480,90 @@
     });
   }
 
+  function initMoveButton() {
+    function initializeJSTree(folders_tree) {
+      var timeOut = false;
+
+      folders_tree.jstree(
+        {
+          "core": {
+            'multiple': false,
+            "themes": {
+              "dots": false,
+              "variant": "large"
+            }
+          },
+          "search": {
+            "show_only_matches": true,
+            "show_only_matches_children": true
+          },
+          "plugins" : [ "wholerow", "search" ]
+        }
+      );
+
+      folders_tree.on('changed.jstree', function (e, data) {
+        destinationFolder = data.instance.get_node(data.selected[0]).id.replace('folder_', '');
+      });
+
+      // Search timeout
+      $('#searchFolderTree').keyup(function () {
+        if(timeOut) { clearTimeout(timeOut); }
+        timeOut = setTimeout(function () {
+          folders_tree.jstree(true).search($('#searchFolderTree').val());
+        }, 250);
+      });
+    }
+
+    function loadMoveToModal(url) {
+      let items;
+      let projects;
+      let folders;
+
+      if ((selectedProjects.length) && (selectedProjectFolders.length)) {
+        items = 'project_and_folders';
+      } else if (selectedProjectFolders.length) {
+        items = 'folders';
+      } else {
+        items = 'projects';
+      }
+      projects = selectedProjects.map(e => ({id: e, type: 'project'}));
+      folders = selectedProjectFolders.map(e => ({id: e, type: 'project_folder'}));
+      let movables = projects.concat(folders);
+
+      $.get(url, {items: items}, function(result) {
+        $(moveToModal).find('.modal-content').html(result.html);
+        $(moveToModal).modal('show');
+        initializeJSTree($(moveToModal).find("#moveToFolders"));
+
+        $(moveToModal).find('form')
+          .on('ajax:before', function() {
+            $('<input>').attr({ type: 'hidden',
+              name: 'destination_folder_id',
+              value: destinationFolder}).appendTo($(moveToModal).find('form'));
+
+            $('<input>').attr({ type: 'hidden',
+              name: 'movables',
+              value: JSON.stringify(movables)}).appendTo($(moveToModal).find('form'));
```

Comment:
try shorter syntax
```
$('<input>', { type: 'hidden', name: param.name, value: param.value})
```

Reply:
for some reason, this is not working... ?